src/index.html

```diff
@@ -5,8 +5,348 @@
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p>Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <div class="card__buybutton">
+          <a href="#" class="card__buylink">
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          class="card__image"
+          alt="cardimage"
+          width="160px"
+          height="134px"
+        >
+
+        <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+        <p class="card__prodcode">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p>Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <div class="card__buybutton">
+          <a href="#" class="card__buylink">
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          class="card__image"
+          alt="cardimage"
+          width="160px"
+          height="134px"
+        >
+
+        <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+        <p class="card__prodcode">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p>Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <div class="card__buybutton">
+          <a href="#" class="card__buylink">
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          class="card__image"
+          alt="cardimage"
+          width="160px"
+          height="134px"
+        >
+
+        <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+        <p class="card__prodcode">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
```

Comment:
```suggestion
        <div class="card__review">
          <div class="card__stars stars stars--4">
```
![image](https://github.com/mate-academy/layout_catalog/assets/59209337/55e31438-206e-4b4e-a8f9-426f9eb6d9be)